.github/actions/build_pandas/action.yml

```diff
@@ -11,12 +11,15 @@ runs:
       shell: bash -el {0}
 
     - name: Build Pandas
+      if : ${{ runner.os != 'Windows' }}
       run: |
-        python setup.py build_ext -j $N_JOBS
-        python -m pip install -e . --no-build-isolation --no-use-pep517 --no-index
+        pip install . --no-build-isolation -v
       shell: bash -el {0}
-      env:
-        # Cannot use parallel compilation on Windows, see https://github.com/pandas-dev/pandas/issues/30873
-        # GH 47305: Parallel build causes flaky ImportError: /home/runner/work/pandas/pandas/pandas/_libs/tslibs/timestamps.cpython-38-x86_64-linux-gnu.so: undefined symbol: pandas_datetime_to_datetimestruct
-        N_JOBS: 1
-        #N_JOBS: ${{ runner.os == 'Windows' && 1 || 2 }}
+
+    - name: Build Pandas (Windows)
+      if: ${{ runner.os == 'Windows' }}
+      run: |
+        call micromamba activate test
+        call "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
```

Comment:
Guessing this is waiting on a meson fix?

Reply:
Yeah, I'll need to find some time to write it.